pkg/common/util/atomic/atomic.go

```diff
@@ -0,0 +1,26 @@
+package atomic
```

Comment:
this seems unused?

Reply:
removing for now, i'll add it back in the subsequent diff where it gets used